### PR TITLE
WIP New functions, dict transform, and lowercase attributes

### PR DESCRIPTION
**Pull request by Mattsface**
This PR adds a new function to MlbDataAdapter that lowers all keys returned from requests.

Also, a function has been added to mlb.py to merge dictionary keys passed to it. 

Classes have been updated to use lowercase attributes as well as tests.

@KCNilssen can you pull this and update the game class? Then we review.

### CODE REVIEW

File: mlbstatsapi/mlbapi.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def get_people_id(self, fullName) -> List[int]:
         # if mlbdata is not empty, and 'people' key is in mlbdata.data and mlbdata.data['people'] is not empty list
         if ('people' in mlbdata.data and mlbdata.data['people']): 
             for person in mlbdata.data['people']:
-                if person['fullName'] == fullName: # Match person fullName
+                if person['fullname'] == fullName: # Match person fullName
```
Comment (KCNilssen) [inline mlbstatsapi/mlbapi.py:78]: A good idea might be to change this to:
`if person['fullname'].lower() == fullName.lower(): # Match person fullName`

File: mlbstatsapi/mlbapi.py [hunk 2]
```diff
@@ -132,7 +132,7 @@ def get_team_id(self, teamName) -> List[int]:
         
         if ('teams' in mlbdata.data and mlbdata.data['teams']):
             for team in mlbdata.data['teams']:
-                if team['teamName'] == teamName: # find matching Team Name
+                if team['teamname'] == teamName: # find matching Team Name
```
Comment (KCNilssen) [inline mlbstatsapi/mlbapi.py:135]: Same thing here:
`if team['teamname'].lower() == teamName.lower(): # find matching Team Name`

File: mlbstatsapi/models/divisions/division.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ﻿from dataclasses import dataclass
 from typing import Optional, Union
+from mlbstatsapi.models import divisions
```
Comment (KCNilssen) [inline mlbstatsapi/models/divisions/division.py:3]: What is the need for this import?

File: mlbstatsapi/mlbapi.py (hunk 1 above)
Comment (Mattsface) [reply mlbstatsapi/mlbapi.py]: changed

File: mlbstatsapi/mlbapi.py (hunk 2 above)
Comment (Mattsface) [reply mlbstatsapi/mlbapi.py]: changed?

Wait I changed it it in the PR?

File: mlbstatsapi/models/divisions/division.py (hunk 3 above)
Comment (Mattsface) [reply mlbstatsapi/models/divisions/division.py]: should be good now

**Comment by Mattsface** [comment]
LGTM Let's ship it @KCNilssen

**Review by KCNilssen — APPROVED** [review]

**Pull request merged**